docker_images/flair/app/pipelines/token_classification.py

```diff
@@ -27,21 +27,30 @@ def __call__(self, inputs: str) -> List[Dict[str, Any]]:
         """
         sentence: Sentence = Sentence(inputs)
 
-        # Also show scores for recognized NEs
-        self.tagger.predict(sentence, label_name="predicted")
+        self.tagger.predict(sentence)
 
         entities = []
-        for span in sentence.get_spans("predicted"):
-            if len(span.tokens) == 0:
-                continue
-            current_entity = {
-                "entity_group": span.tag,
-                "word": span.text,
-                "start": span.tokens[0].start_position,
-                "end": span.tokens[-1].end_position,
-                "score": span.score,
-            }
-
-            entities.append(current_entity)
+        for label in sentence.get_labels():
```

Comment:
Could you please add a test of a valid PoS model as well in https://github.com/huggingface/api-inference-community/blob/fe8ceac7afeb9bd376f5f6d6b233d7e116cdc5e3/docker_images/flair/tests/test_api.py#L11 ? You can check the sentence-transformers codebase to support multiple models

Reply:
Thanks for the hint, I've also introduced parametrized tests here and fixed a bug for the `test_unsupported_tasks`  case.

`setUpClass` was also necessary - otherwise I didn't see the output ("SequenceTagger predicts") from second model.